cmd/posix_test.go

```diff
@@ -35,7 +36,13 @@ func newPosixTestSetup() (StorageAPI, string, error) {
 	if err != nil {
 		return nil, "", err
 	}
-	posixStorage, err := newPosix(diskPath)
+	var endpoints []*url.URL
+	// Initialize a new posix layer.
+	endpoints, err = parseStorageEndpoints([]string{diskPath})
+	if err != nil {
+		return nil, "", err
+	}
+	posixStorage, err := newPosix(endpoints[0])
```

Comment:
even for remote rpc disk, the stack will look like:
fsObjects->rpcClient->rpcServer->posix

here rpcServer just needs to do newPosix(path) instead of newPosix(url)